plugins/mcreator-localization/lang/texts.properties

```diff
@@ -733,7 +733,9 @@ blockly.category.directionactions=Direction procedures
 blockly.category.entityprocedures=Entity procedures
 blockly.category.entitydata=Data
 blockly.category.entitymanagement=Actions
-blockly.category.itemmanagement=Item procedures
+blockly.category.itemdata=Item data
+blockly.category.itemmanagement=Item management
```

Comment:
Other categories just use "Data/Actions" now instead management and duplicating the category type

Reply:
My brain still needs to be updated 🤣

Reply:
Unless there is caching issue on github, I still don't see data / actions

Reply:
Not cahcing issue, laptop problem. I was using the online editor on my laptop and it didn't save (while I was seeing them as changed)

Reply:
Now they are properly changed